app/views/course/user_invitations/_invitation_fields.html.slim

```diff
@@ -3,7 +3,7 @@
   td = f.input :email, required: true, label: false, input_html: { class: 'invitation_email' }
   td = f.input :role,
                as: :select,
-               collection: Course::UserInvitation.roles.keys,
+               collection: Course::UserInvitation.invitable_roles(current_course_user.role).keys,
```

Comment:
This fails for super admins as their `current_course_user` might be `nil` since they do not belong to the course.